.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature
+about: New capability that delivers business outcome
+title: Title
+labels: Feature
+assignees: ''
+
+---
+## 💬 Brief description of feature
+
+<!-- In a few words, what is this feature -->
+
+## ☑️ Business impact
+<!-- detail how business will benefit from this feature -->
```

Comment:
This content ensures clients are aware of how the feature is going to help the organisation.

I/we should also focus on trying to ensure that benefits are delivered.  Most of the client interactions are through me, so it's a reminder to me that our job isn't finished with the release of a new version - it's delivery of the business benefit.